lib/template-parts.php

```diff
@@ -138,7 +138,7 @@ function filter_rest_wp_template_part_query( $args, $request ) {
 		$tax_query   = isset( $args['tax_query'] ) ? $args['tax_query'] : array();
 		$tax_query[] = array(
 			'taxonomy' => 'wp_theme',
-			'field'    => 'slug',
+			'field'    => 'name',
```

Comment:
Im curious about the switch from `slug` to `name` here?  It looks like the [templates-sync ](https://github.com/WordPress/gutenberg/blob/fe5bd763b3df661982c4c713f6d18016d3fa1768/lib/templates-sync.php#L33-L48)still queries and creates based on `slug`, so could this potentially cause issues?

Reply:
> Im curious about the switch from slug to name here?

When themes are added to taxonomies, a slug is generated that looks like `parent-directory-some-theme` while the name looks like `parent-directory/some-theme`

Using name instead of slug allows us to query for the theme on the client without doing any string transformations. They can simply pass through the value of `getCurrentTheme().stylesheet`

Reply:
> It looks like the templates-sync still queries and creates based on slug, so could this potentially cause issues?

Ahhh I must've missed a few values. But yeah. Ideally all queries with `slug` are switched to name if we go down this route.

Reply:
That all makes sense.  Im not really sure if there are any other implications to using `name` over `slug`.  But if we are consistent with it and it works, it sounds great to have the tax term and `getCurrentTheme().stylesheet` in sync.